### ISSUE TITLE
can we use sdxl lora?

### ISSUE THREAD

**Issue by zzddwyff** [RedAIGC/StoryMaker#25]

**Comment by RedAIGC** [comment]
Yes, you can use lora by "pipe.load_lora_weights(ckpt_path, adapter_name=model_name)/pipe.set_adapters([model_name], adapter_weights=[lora_w])", but the results may not be so good because there is already a lora in StoryMaker model.

**Comment by fatpandaria** [comment]
So what does the existing lora in StoryMaker do? I am just wondering whether I can control the image generation output style by changing the base model and loras; @RedAIGC
